ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -6,8 +6,9 @@
 
 
 def add_user(*args):
-    User = get_user_model()
-    User(serial='__system__', full_name='System', password='!!').save()
+    queryset = get_user_model().objects.get_queryset_unfiltered()
+    queryset.update_or_create(serial='__system__',
+                              defaults={'full_name':'System', 'password':'!!'})
```

Comment:
I have a slicy preference to keep `get_queryset_unfiltered()` but it is not so important.
Whatever what we keep we should move to keep only one way to do is better.